src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/bucketize.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/bucketize.hpp"
+
+#include <vector>
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<std::vector<size_t>> data_shapes = {
+    // No reason to test other ranks as logic is the same
+    {40, 22, 13, 9},     // 4D
+    {6, 7, 3, 2, 8, 5},  // 6D
+};
+
+const std::vector<std::vector<size_t>> buckets_shapes = {
+    {5},
+    {20},
+    {100},
+};
+
+const std::vector<bool> with_right_bound = {true, false};
+
+const std::vector<InferenceEngine::Precision> out_precision = {
+    InferenceEngine::Precision::I32,
+    InferenceEngine::Precision::I64,
+};
+
+// We won't test FP32 and FP16 together as it won't make sense for now
+// as ngraph reference implementation use FP32 for FP16 case
+
+INSTANTIATE_TEST_SUITE_P(smoke_Bucketize_input_fp16,
```

Comment:
Could you please add i8 and u8 test cases if available?

Reply:
No, I cannot add them as they are not supported right now in ngraph reference implementation. We would need to enable those types in "docs/template_plugin/backend/evaluates_map.cpp" file for "op::v3::Bucketize". But I added cldnn unit tests for those types.

Reply:
I added support for i8 and u8 for bucketize operation in ngraph part, and also in my GPU func tests.